index.html

```diff
@@ -78,11 +98,12 @@ <h2>I'm a software developer</h2>
         <section class="works">
           <div class="heading-works">
             <h1>My recent works</h1>
+            <div class="underline"></div>
           </div>
 
           <div class="work-details">
             <div class="work-1">
-              <div class="work-detail">
+              <div class="work-1__detail">
                 <img
                   src="./assets/images/Img Placeholder.png"
                   alt="multi-post stories"
```

Comment:
- [ ] [REQUIRED] Good job so far! Please correct the `position` of the elements in the section mentioned in the `image` below and please implement correct style for `button`(Hover, Pressed, and `border-radius`) like [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=kLoX8FBKAxK9lEJJ-0)🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/220958472-69aa50d5-0c32-4c2e-9f8b-74d9d58f9a7b.png)